src/openrct2-ui/windows/News.cpp

```diff
@@ -391,7 +392,8 @@ namespace OpenRCT2::Ui::Windows
         ScreenSize onScrollGetSize(int32_t scrollIndex) override
         {
             int32_t scrollHeight = static_cast<int32_t>(getGameState().newsItems.GetArchived().size())
-                * CalculateNewsItemHeight();
+                    * CalculateNewsItemHeight()
+                - 1;
```

Comment:
What's the reason for the -1 here? (I only noticed because of the wonky indentation, tbh.)

If it's to offset a final separator line, it might be worth adding a `kSeparatorHeight` constant.

Reply:
The separator is bigger than that (being two lines), but for some reason there an additional line of crud at the bottom that wasn’t there before. Couldn’t find any other way.

Reply:
Hmm. Could it just be drawing a separator that it wasn't drawing before?